src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,38 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
 
+    private String brand;
+    private String model;
+    private String color;
+    private String bodyType;
+    private int win;
+    private int yearOfIssue;
+    private Engine engine;
+
+    public Car(String brand, String model, String color, String bodyType,
+               int win, int yearOfIssue) {
+        this.brand = brand;
+        this.model = model;
+        this.color = color;
+        this.bodyType = bodyType;
+        this.win = win;
+        this.yearOfIssue = yearOfIssue;
+    }
+
+    @Override
+    public Car clone() {
+        Car clonedCar = new Car(this.brand, this.model, this.color,
+                this.bodyType, this.win, this.yearOfIssue);
+        clonedCar.engine = engine.clone();
```

Comment:
try to use `engine.clone()` in the Car constructor

Reply:
ok